test/controllers/login_controller_test.rb

```diff
@@ -29,7 +29,6 @@ class LoginControllerTest < ActionController::TestCase
 
     describe "#urs callback" do
       it "should successfully create a user" do
-        skip "Creating a new user. Todo: figure out transaction rollbacks."
```

Comment:
We can no remove this skipped test, as rails 5 has better support for cleaning up transactions after each test, seems to work fine now.